packages/transition-backend/src/api/services.socketRoutes.ts

```diff
@@ -194,35 +191,18 @@ export default function (socket: EventEmitter, userId?: number) {
     if (userId !== undefined) {
         socket.on(
             TrRoutingConstants.BATCH_ROUTE,
-            async (
-                parameters: TransitBatchRoutingDemandAttributes,
-                transitRoutingAttributes: BatchCalculationParameters,
-                callback
-            ) => {
+            async (parameters: CsvFileAndMapping, transitRoutingAttributes: BatchCalculationParameters, callback) => {
```

Comment:
I think we should have something properly defined for the CsvMapping, having a specific type which define which field we expect in the CSV file at this point. So we know we have all the right field already. 
We are losing the definition of the parameters that were in TransitBatchRoutingDemandAttributes